src/templates/finops-hub/modules/infrastructure.bicep

```
@@ -19,7 +19,7 @@ param hub HubProperties
 var safeHubName = replace(replace(toLower(hub.name), '-', ''), '_', '')
 // cSpell:ignore vnet
 var vNetName = '${safeHubName}-vnet-${hub.location}'
-var nsgName = '${hub.routing.networkName}-nsg'
+var nsgName = '${vNetName}-nsg'
```

I would not recommend this change. The hub.routing.networkName property is available to all apps. We should use that as the source of truth. If that's the same as vNetName here, then remove vNetName.
```suggestion
var nsgName = '${hub.routing.networkName}-nsg'
```